src/main/interface/cli.c

```diff
@@ -927,6 +927,15 @@ static void cliSerial(char *cmdline)
 }
 
 #ifndef SKIP_SERIAL_PASSTHROUGH
+#ifdef USE_PINIO
+static void cbCtrlLine(void *context, uint16_t ctrl)
+{
+    int pinioDtr = (int)(long)context;
+
+    pinioSet(pinioDtr, ~ctrl & CTRL_LINE_STATE_DTR);
```

Comment:
`pinioSet(pinioDtr, !(ctrl & CTRL_LINE_STATE_DTR));` is easier to understand ...

Reply:
OK